### ISSUE TITLE
Accessibility violations with `Link` component

### ISSUE THREAD

**Issue by chaitanyadeorukhkar** [razorpay/blade#1000]
Ref: https://blade.razorpay.com/?path=/story/components-link--default&globals=measureEnabled:false

<img width="1169" alt="Screenshot 2023-02-09 at 11 30 42 AM" src="https://user-images.githubusercontent.com/24487274/217730788-8c8e7a93-75f8-45c3-af59-cd5e0b9542fd.png">

**Comment by chaitanyadeorukhkar** [comment]
@saurav12 can you take a look at this issue?

**Comment by saurav12** [comment]
@chaitanyadeorukhkar This is more than the least required ratio. We support a minimum of `3:1` and it is coming out to be `3:31`.

**Comment by anuraghazra** [comment]
WCAG2AA requires `4.5:1` ratio for small text & `3:1` for large text, thus the above check is failing.

**Comment by adilk121** [comment]
Combine Adjacent Links into a Single Link

**Comment by anuraghazra** [comment]
This violation has been fixed after rebranding.
